.github/ISSUE_TEMPLATE/support.yml

```
@@ -0,0 +1,60 @@
+## Description
+
+name: Bug Report
+description: Request support or help from the Valet team
+title: "[Support]: "
+labels: ["help wanted"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide the following information to help us troubleshoot your issue.
+  - type: checkboxes
+    id: os
+    attributes:
+      label: What operating system are you using?
+      options:
+        - label: macOS
+        - label: Linux
+        - label: Windows
+  - type: textarea
+    id: gh-version
+    attributes:
+      label: What version of the `gh` CLI are you using?
+      placeholder: i.e. the output of `gh version`
+    validations:
+      required: true
+  - type: textarea
+    id: gh-version
+    attributes:
+      label: What version of the `gh-valet` extension are you using?
+      placeholder: i.e. the output of `gh extension list`
```

```suggestion
      placeholder: i.e. the output of `gh extension list | grep valet`
```
Should we give a more specific command so they have less to dig through?

---

Love that!